devel/binutils/Portfile

```diff
@@ -28,7 +28,8 @@ configure.args      --infodir='${prefix}/share/info' \
                     --mandir='${prefix}/share/man' \
                     --disable-werror \
                     --program-prefix=g \
-                    --enable-shared
+                    --enable-shared \
+		    --enable-install-libbfd
```

Comment:
Minor whitespace fix would be welcome here.

Reply:
That’s what I get for doing that outside of Xcode

Reply:
There are still tabs instead of spaces.

Reply:
Yeah your right, redone that now it’s spaces